css/style.css

```diff
@@ -0,0 +1,486 @@
+body {
+  font-size: 0.8rem;
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+hr {
+  border: 1px solid #e3e3e3;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* flex box */
+.flex{
+    display: flex;
+}
+
+.nav-bar {
+ 
+  flex-direction: row;
+  justify-content: space-between;
+  height: 36px;
+  margin: 20px;
+}
+
+.col1 {
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-right: 15%;
+}
+
+.col1 > .logo {
+  width: 80px;
+  height: 24px;
+  margin-top: 10px;
+}
+
+.col2 {
+  flex-direction: row;
+  flex: 0.7;
+  margin-top: 10px;
+  justify-content: center;
+}
+
+.col2 > form {
+  flex: 1;
+  height: 32px;
+}
+
+.col2 .searchbox {
+  margin-right: 0;
+  padding-right: 0;
+  flex-grow: 0.7;
+  border-style: solid;
+  border-radius: 0;
+  border-right: 0;
+  box-shadow: none !important;
+}
+
+.col2 .searchBtn {
+  margin-left: 0 !important;
+  border-style: solid;
+  border-radius: 0;
+  cursor: pointer;
+}
+
+.comments-icon > svg {
+  width: 24px;
+  height: 24px;
+
+  /* margin-right: 10px; */
```

Comment:
[OPTIONAL] It's optional since it wasn't pointed earlier, but I strongly suggest you guys to implement it.
- [ ] Comments about the code are allowed, commented code not. Please take care of it boys 💪